src/harness/unittests/matchFiles.ts

```diff
@@ -109,23 +109,21 @@ namespace ts {
         }
         {
             const actual = ts.parseJsonConfigFileContent(json, host, basePath, existingOptions, configFileName, resolutionStack);
-            expected.errors = map(expected.errors, error => {
-                return <Diagnostic>{
-                    category: error.category,
-                    code: error.code,
-                    file: undefined,
-                    length: undefined,
-                    messageText: error.messageText,
-                    start: undefined,
-                };
-            });
+            expected.errors = expected.errors.map<Diagnostic>(error => ({

```

Comment:
My personal preference is to annotate the return type on the lambda (`(error): Diagnostic => ...`), rather than specifying the generic argument. Specifying generic arguments always feels a tad clunky to me. No big deal either way, though.